docs/rules/template-tag-spacing.md

```diff
@@ -0,0 +1,70 @@
+# Enforce Usage of Spacing in Tagged Template Literals (template-tag-spacing)
+
+(fixable) The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems reported by this rule.
+
+We can create functions whose parameters consist of template literals.
```

Comment:
Yeah, you're right. I rewrote the whole first paragraph and added a "Further reading" section to explain the whole concept a bit better.